Salesforce Org Setup/force-app/main/default/lwc/incidentMap/incidentMap.js

```diff
@@ -0,0 +1,44 @@
+import {
+  LightningElement,
+  wire,
+  api,
+  track
+} from 'lwc';
+import {
+  getRecord
+} from 'lightning/uiRecordApi';
+
+const FIELDS = [
+  // 'Case.Incident_Location__c',
+  'Case.Incident_Location__Latitude__s',
+  'Case.Incident_Location__Longitude__s'
+];
+
+export default class LightningExampleMapSingleMarker extends LightningElement {
+  @api recordId;
+  @track mapMarkers;
+  @track error;
+  @wire(getRecord, {
+    recordId: '$recordId',
+    fields  : FIELDS
```

Comment:
Rename const to `fields`, so you can just have `fields` instead of `fields: fields`.